controller/src/lib.rs

```diff
@@ -38,8 +38,8 @@ pub async fn run_scheduler(nats: TypedNats) -> NeverResult {
                 }
             },
 
-            spawn_request = spawn_request_sub.next() => {
-                match spawn_request {
+            schedule_request = schedule_request_sub.next() => {
+                match schedule_request {
                     Some(schedule_request) => {
                         tracing::info!(spawn_request=?schedule_request.value, "Got spawn request");
```

Comment:
we might not want to log these now that they have the tokens in them, or filter out the token before logging?

Reply:
my b got confused.